quad_utils/rviz/example_with_terrain.rviz

```diff
@@ -454,6 +456,6 @@ Window Geometry:
     collapsed: false
   Views:
     collapsed: true
-  Width: 1440
-  X: 1920
-  Y: 0
+  Width: 1613
+  X: 67
+  Y: 27
```

Comment:
Revert all these changes except switching the state topic name/tf prefix

Reply:
sounds good